src/components/Navigation.js

```diff
@@ -169,22 +213,196 @@ function Navigation() {
           <div className="d-flex flex-row align-items-center ms-auto navigation-search">
             <div className="d-flex flex-row align-items-center">
               <input
-                className="form-control me-2"
+                className="form-control me-2 bg-dark text-light navigation-search-input"
                 type="search"
-                placeholder="Search tx or address"
+                placeholder={`Search here`}
                 aria-label="Search"
                 ref={txSearchRef}
                 onKeyUp={handleKeyUp}
               />
-              <i className="fa fa-search pointer" onClick={search}></i>
             </div>
-            <Version />
           </div>
         </div>
+        <div className="mainnet-container hide-tabs">
+          <img
+            src={theme === 'dark' ? sun : moon}
+            alt="themeColorButton"
+            className="theme-color-btn"
+            onClick={() => dispatch(toggleTheme())}
+          />
```

Comment:
suggestion: Move this to the theme toggle component.

Maybe the theme toggle component should receive a bool to render as a switch or an image button.
This way we don't have to inline the theme toggle.

Reply:
I refactored the JSX to make the roles of the elements clearer. Could you please review it?